torchci/lib/types.ts

```diff
@@ -21,8 +21,8 @@ export interface JobData extends BasicJobData {
   logUrl?: string;
   durationS?: number;
   queueTimeS?: number;
-  failureLine?: string;
-  failureLineNumber?: number;
+  failureLines?: string[];
+  failureLineNumbers?: number[];
```

Comment:
I think we might want to consider updating `RecentWorkflowsData` as well for consistency. It also has a single `failure_line` string field.

For the context, this failure line field is pretty important because it now influences how Dr.CI and mergebot detects flaky failures (I search for similar failures using this field as the input)

Reply:
May be it's better to do this in a separate PR, as this one is pretty big already.